setup/updating.md

```diff
@@ -37,12 +37,20 @@ Updating Coder is a two-step process:
     ```bash
     helm repo update
     ```
+    
+2. Export the current helm chart values into a file:
 
-2. Upgrade to the desired version (i.e., `1.16.0`):
+    ```bash
+    helm get values --namespace coder coder > current-values.yml
+    ```
+
+3. Upgrade to the desired version with your helm chart values file (i.e., `1.16.0`):
+
+    _Note: Omitting `--version` will default to the latest version_
 
     ```bash
     helm upgrade --namespace coder --force --install --atomic --wait \
-      --version 1.16.0 coder coder/coder
+      --version 1.16.0 coder coder/coder -f current-values.yml
```

Comment:
version should be 1.16.1 now

Also my preference is to use `--values` instead of `-f` (long commands are always preferable to make what the commands are doing more obvious)

Reply:
Yeah, we need to consider how we want to handle versions probably @khorne3. Updating the docs for every single patch release is going to be a pain in terms of technical debt.

Agreed on `--values` which means I'll update the doc section below this too